daprdocs/content/en/js-sdk-docs/_index.md

```diff
@@ -9,15 +9,45 @@ no_list: true
 
 The Dapr JS SDK will allow you to interface with the Dapr process that abstracts several commonly used functionalities such as Service-to-Service invocation, State Management, PubSub, and more.
 
+## Installation
+
+To get started with the Javascript SDK, you can download the Dapr Javascript SDK package from [NPM](https://npmjs.org/package/dapr-client) by running the following:
+
+```bash
+npm install --save dapr-client
+```
+
+## Structure
+
+The Dapr Javascript SDK contains two major components: 
+
+* **DaprServer:** The Dapr Server manages all communication from the Dapr Sidecar to your application
+* **DaprClient:** The Dapr Client manages all communication from your application to the Dapr Sidecar
+
+The above communication can be configured to use either of the gRPC or HTTP protocols.
+
+![Dapr Server](./js-server/dapr-server.jpg)
```

Comment:
The images look slightly oversized in the preview, do you want to constraint it to a specific size? https://github.com/dapr/js-sdk/blob/aa14ebf406fb0ac376d6306971503a244011d771/daprdocs/content/en/js-sdk-docs/_index.md